<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KexiPasswordWidget</class>
 <widget class="QWidget" name="KexiPasswordWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>174</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="prompt">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>10</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Supply a password below.</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="KTitleWidget" name="errorMessage"/>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="userNameLabel">
       <property name="text">
        <string>Username:</string>
       </property>
       <property name="buddy">
        <cstring>userEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KLineEdit" name="userEdit">
       <property name="showClearButton" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="anonymousLabel">
       <property name="text">
        <string>Anonymous:</string>
       </property>
       <property name="buddy">
        <cstring>anonymousCheckBox</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="anonymousCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Database name:</string>
       </property>
       <property name="buddy">
        <cstring>nameEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="KLineEdit" name="nameEdit">
       <property name="showClearButton" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="domainLabel">
       <property name="text">
        <string>Database server:</string>
       </property>
       <property name="buddy">
        <cstring>domainEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="KLineEdit" name="domainEdit">
       <property name="showClearButton" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>Password:</string>
       </property>
       <property name="buddy">
        <cstring>passEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="KLineEdit" name="passEdit">
       <property name="showClearButton" stdset="0">
        <bool>true</bool>
       </property>
       <property name="passwordMode">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="keepCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="keepCheckBoxLabel">
       <property name="text">
        <string>Remember password:</string>
       </property>
       <property name="buddy">
        <cstring>keepCheckBox</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTitleWidget</class>
   <extends>QWidget</extends>
   <header>ktitlewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>userEdit</tabstop>
  <tabstop>anonymousCheckBox</tabstop>
  <tabstop>domainEdit</tabstop>
  <tabstop>passEdit</tabstop>
  <tabstop>keepCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
